lib/components/Pagination/Pagination.tsx

```diff
@@ -94,6 +96,17 @@ const Page = ({ number, current }: { number: number; current: boolean }) => {
         borderRadius="standard"
         className={[styles.background, current ? styles.current : undefined]}
       />
+      <Overlay
+        component="span"
+        transition={current ? undefined : 'fast'}
```

Comment:
Not sure this should have a transition? Looks like my screen is ghosting or something.

Reply:
```suggestion
```